pyproject.toml

```
@@ -5,6 +5,9 @@ description = "Python client for Apple WeatherKit API"
 authors = ["TJ Horner <me@tjhorner.com>"]
 readme = "README.md"
 repository = "https://github.com/tjhorner/python-weatherkit"
+classifiers = [
+  "License :: OSI Approved :: MIT License",
+]
```

The project uses `poetry`.

```suggestion
license = "MIT"
```
https://python-poetry.org/docs/pyproject/#license